packages/math-input/src/components/tabbar/tabbar.tsx

```diff
@@ -19,41 +19,29 @@ const styles = StyleSheet.create({
     },
 });
 
-type TabbarState = {
-    selectedItem: number;
-};
-
 type Props = {
     items: ReadonlyArray<TabbarItemType>;
-    onSelect: (item: TabbarItemType) => void;
+    selectedItem: TabbarItemType;
+    onSelectItem: (item: TabbarItemType) => void;
 };
 
-class Tabbar extends React.Component<Props, TabbarState> {
-    state: TabbarState = {
-        selectedItem: 0,
-    };
-    render(): React.ReactNode {
-        const {items, onSelect} = this.props;
-        return (
-            <View style={styles.tabbar}>
-                {items.map((item, index) => (
-                    <TabbarItem
-                        key={`tabbar-item-${index}`}
-                        itemState={
-                            index === this.state.selectedItem
-                                ? "active"
-                                : "inactive"
-                        }
-                        itemType={item}
-                        onClick={() => {
-                            this.setState({selectedItem: index});
-                            onSelect(item);
-                        }}
-                    />
-                ))}
-            </View>
-        );
-    }
+function Tabbar(props: Props): React.ReactElement {
```

Comment:
I know arrows are in vogue, but I much prefer `function` myself. Not a hill I'd want to die on, but I think I'd rather read two styles than have another linting rule yelling at me. If it could be auto-fixed then I'd be onboard 100%, but it seems like something that might be dangerous to auto-correct.